cli/cmd/pool.go

```diff
@@ -319,3 +326,37 @@ func (c *ServicedCli) cmdRemoveVirtualIP(ctx *cli.Context) {
 		fmt.Printf("Removed virtual IP: %v from pool %v\n", args[1], args[0])
 	}
 }
+
+// serviced pool set-conn-timeout POOLID TIMEOUT
+func (c *ServicedCli) cmdSetConnTimeout(ctx *cli.Context) {
+	args := ctx.Args()
+	if len(args) != 2 {
+		fmt.Printf("Incorrect Usage.\n\n")
+		cli.ShowCommandHelp(ctx, "set-conn-timeout")
+		return
+	}
+
+	connTimeout, err := time.ParseDuration(args[1])
```

Comment:
We should have some cli tests to verify that negative numbers are rejected, and that 0 or positive numbers are accepted as valid.